Library/Homebrew/cmd/update.sh

```diff
@@ -2,8 +2,11 @@
 #:
 #:  Fetch the newest version of Homebrew and all formulae from GitHub using `git`(1) and perform any necessary migrations.
 #:
-#:       --merge                         `git merge` is used to include updates (rather than `git rebase`).
-#:       --force                         Always do a slower, full update check (even if unnecessary).
+#:          --merge                      `git merge` is used to include updates (rather than `git rebase`).
+#:      -f, --force                      Always do a slower, full update check (even if unnecessary).
+#:      -v, --verbose                    Print the directories checked and `git` operations performed.
+#:      -d, --debug                      Display any debugging information.
+#:      -h, --help                       Show this message.
```

Comment:
This can be skipped as we're not documenting in on other commands.

Reply:
This is documented in the `Homebrew::CLI::Parser.global_options`, so it does get listed in the `brew help` output for every command. You're right though it does not get pulled in when generating the man page markdown. Is it more important to exclude this for consistency in the man page or include it for consistency at the shell command line?

Reply:
Ooh, nice spot. Ideally the manpage and help output would be consistent regardless for all commands (i.e. display `--help` in both places or neither). I'm easy on either approach. Could you add that in this PR or a separate one? I don't mind which. Thanks!

Reply:
@MikeMcQuaid It looks like man.rb was designed with the intent of documenting [global options](https://docs.brew.sh/Manpage#global-options) separately from "command-specific options", so that's why `--help` does not appear in each command usage. I don't really want to mess with that organization. But all options do and should appear at the command line.

So how about a change that just filters out `--help` with a regex from shell command when building the man page?

Reply:
> So how about a change that just filters out `--help` with a regex from shell command when building the man page?

Sounds good if that's the only inconsistency 👍

Reply:
In 88dea00 I added a regex to filter out all of the "global options" from shell command usage blocks. This is now consistent with the behavior of ruby cmds. All options are printed on the command line with `brew help update`, but the man page and html only lists the `--merge` option since all other options are in the "global options" set, including `--force`. Hope this makes sense.